### PR TITLE
Quest: A Timely Visit

### PR DESCRIPTION
**Pull request by beauxq**

### CODE REVIEW

File: scripts/zones/Southern_San_dOria/npcs/Deraquien.lua [hunk 1]
```diff
@@ -54,23 +92,36 @@ end;
 -----------------------------------
 
 function onEventFinish(player,csid,option)
-    -- printf("CSID: %u",csid);
-    -- printf("RESULT: %u",option);
+    --printf("CSID: %u",csid);
```
Comment (TeoTwawki) [inline scripts/zones/Southern_San_dOria/npcs/Deraquien.lua]: dun do that. Also I see tabs. Configure your editor to convert tabkey presses into 4 spaces. And/or/also get editor config installed for it to automate this for you  (most editors have a plugin)

**Comment by beauxq** [comment]
made a mistake with this branch, I'll make new pull request.

**Comment by takhlaq** [comment]
no need to delete/close, can squash/rebase and force push

**Comment by beauxq** [comment]
That's what I was trying to do, but I accidentally squashed too many commits. Then trying to undo that, something else messed up. So it was easier to just delete, close, and make a new one.
https://github.com/DarkstarProject/darkstar/pull/4107

**Pull request closed without merging**